src/i18n/en-us/help.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  noAddress:
+    'In order to see all help requests in your neighbourhood, you need to set your address in the settings.',
```

Comment:
"To see specific requests for help from your neighbourhood, you can enter your address here or save your address in the profile settings."

Generell ist es eine gute Idee "need" or "müssen" zu vermeiden.